test/inductor/test_compiled_optimizers.py

```diff
@@ -160,7 +160,7 @@ def tearDown(self):
     test_adamw = make_test(AdamW, lr=0.01)
     # Need to an impl which does not use python scalars
     # test_adamax = make_test(Adamax, lr=0.01)
-    # test_nadam = make_test(NAdam, lr=0.01)
+    test_nadam = make_test(NAdam, lr=0.01)
```

Comment:
Could you add more variants here similar to the configs in https://github.com/pytorch/pytorch/blob/main/test/optim/test_optim.py#L841-L849